app/jobs/connect_app_job.rb

```diff
@@ -1,15 +1,28 @@
 # frozen_string_literal: true
 
 class ConnectAppJob < ApplicationJob
-  def perform(app, token)
+  include CableReady::Broadcaster
+
+  def perform(app, token, channel: nil)
     url = "https://n8n.openfoodnetwork.org.uk/webhook/regen/connect-enterprise"
     event = "connect-app"
+    enterprise = app.enterprise
     payload = {
-      enterprise_id: app.enterprise_id,
+      enterprise_id: enterprise.id,
       access_token: token,
     }
 
     response = WebhookDeliveryJob.perform_now(url, event, payload)
     app.update!(data: JSON.parse(response))
+
+    return unless channel
+
+    selector = "#edit_enterprise_#{enterprise.id} #connected_apps_panel div"
```

Comment:
If you have multiple tabs open with other enterprises then you don't want to show this enterprise's information on another enterprise's page.